enumerable.rb

```diff
@@ -0,0 +1,117 @@
+# frozen_string_literal: true
+
+module Enumerable
+  def my_each(arr)
+    x = 0
+    while x < arr.length
+      yield(arr[x])
+      x += 1
+    end
+  end
```

Comment:
- [ ]  Enumerables my_each calls the given block once for each element in self
`[1,2,3,4,5].my_each{ |num| puts num }` gives `error`

- [ ] Enumerables my_each returns an Enumerator if no block is given
`[1,2,3,4,5].my_each` should return `#<Enumerator: [1, 2, 3, 4, 5]:my_each>`